code/controllers/subsystem/processing/quirks.dm

```diff
@@ -62,6 +62,9 @@ PROCESSING_SUBSYSTEM_DEF(quirks)
 		// SKYRAT EDIT ADDITION START
 		if(initial(quirk_type.erp_quirk) && CONFIG_GET(flag/disable_erp_preferences))
 			continue
+		// Hidden quirks aren't visible to TGUI or the player
+		if (initial(quirk_type.hidden_quirk))
+			continue;
```

Comment:
```suggestion
			continue
```
go back to your C-languages, `;`.

Reply:
Oops, good catch of my rookie mistake 😛

Reply:
Fixed in commit ccc1de5255e605a62e655a7ed579b757e34b5e98